buerno/bruno.json

```diff
@@ -0,0 +1,5 @@
+{
+  "version": "1",
+  "name": "Bruno",
+  "type": "collection"
+}
```

Comment:
It's a good practice to end a file with a newline character. This can prevent potential issues with some tools that might not handle files without a trailing newline properly. [medium]